src/ui/header/navigation.tsx

```
@@ -0,0 +1,172 @@
+  _id: string;
+  _rev: string;
+  _createdAt: string;
+  _updatedAt: string;
+}
+
+interface Category {
+  title: string;
+  links?: Link[];
+}
+
+interface Link {
```

Biome (1.9.4)

[error] Shouldn't redeclare 'Link'. Consider to delete it or rename it.

'Link' is defined here:


(lint/suspicious/noRedeclare)